addons/google_recaptcha/views/res_config_settings_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.web.recaptcha</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="base_setup.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <div id="recaptcha_warning" position="replace">
+                <div attrs="{'invisible': [('module_google_recaptcha', '=', False)]}">
+                    <div class="content-group mt16">
+                        <label for="recaptcha_public_key" class="o_light_label"/>
+                        <field name="recaptcha_public_key"/>
+                    </div>
+                    <div class="content-group">
+                        <label for="recaptcha_private_key" class="o_light_label"/>
+                        <field name="recaptcha_private_key"/>
+                    </div>
+                    <div class="content-group">
+                        <label for="recaptcha_min_score" class="o_light_label"/>
+                        <field name="recaptcha_min_score"/>
+                    </div>
+                    <div>
+                        <a href="https://www.google.com/recaptcha/admin/create" class="oe_link" target="_blank">
+                            <i class="fa fa-arrow-right"/> Generate public/private keys
```

Comment:
rename label site/secret and add ~'for v3 reCaptcha type' to avoid doubt between v2 and v3